lib/madness/breadcrumbs.rb

```diff
@@ -27,7 +27,7 @@ def get_breadcrumbs
     def breadcrumbs_maker(partial_path)
       parent, basename = File.split partial_path
       item = OpenStruct.new({
-        label: basename.to_label,
+        label: File.basename(basename, '.md').to_label,
```

Comment:
Maybe this would be better in `StringRefinements#to_label`?

Reply:
Ok. You are right